coralproject/talk
# Create a notifications tray that is always available

## What does this PR do?

Adds a live notifications tray that is always available.

- when in desktop mode, it floats as an icon in the top right
- when in mobile mode, it resides in the `Next unread` tray

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None.

## Does this PR introduce any new environment variables or feature flags?

None

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- post a comment
- have another account reply to it
- see the live button icon update with a new count
- click on the button
- see the notification list appear
- scroll through it and load more (if you have enough notifications)
- click on the button to close the list

## Were any tests migrated to React Testing Library?

None

## How do we deploy this PR?

Merge into epic branch and release that.
